book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Well done with your Book class. As part of the requirements, you are supposed to create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. So, Kindy Implements the has-many relation in Book by creating a method to add rentals in the Book class.

Reply:
Needs attention!